2216/2/bad_words_table.rb

```diff
@@ -0,0 +1,88 @@
+class BadWordsTable
```

Comment:
Just for your information: there's a great gem called `terminal-table` which allows you to pretty print tables in your terminal.
Using the gem will reduce the amount of ugly code which does manual formatting.
https://github.com/tj/terminal-table